Resources/Prototypes/DeltaV/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -57,3 +57,79 @@
         Piercing: 0.85
         Heat: 0.75
   - type: AllowSuitStorage
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterStasecCoat
+  name: station security winter coat
+  description: A heavy jacket made from 'synthetic' animal furs, enough to warm even the harshest security officer's cold heart.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/OuterClothing/WinterCoats/coatsec.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/OuterClothing/WinterCoats/coatsec.rsi
+  - type: Armor # Standardised armoured winter coat
+    modifiers:
+      coefficients:
+        Blunt: 0.85
+        Slash: 0.85
+        Piercing: 0.85
+        Heat: 0.75
+  - type: AllowSuitStorage
+
+- type: entity
```

Comment:
Why is this a duplicate of ClothingOuterStasecCoat?

Parent it to Station Security Winter Coat or make a new abstract parent and parent it to that.

Reply:
Consider it done.